gis/centreline/sql/create_view_routing_centreline_directional.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE VIEW gis_core.routing_centreline_directional AS
+
+SELECT
+    centreline.centreline_id,
+    concat(centreline.centreline_id, 0)::integer AS id,
```

Comment:
Unfortunately this isn't like the HERE `link_id`

![image](https://github.com/CityofToronto/bdit_data-sources/assets/10623872/90b9c147-6976-40de-94b6-5b6654d12b0d)

If we need these to be unique then we could instead add 100,000,000 and 200,000,000 to the IDs

Reply:
gahhh maybe I will just row_number it 🤔 edge id changing through out version should be ok right?